packages/browser/src/backend.ts

```diff
@@ -67,13 +67,8 @@ export interface BrowserOptions extends Options {
 
 /** The Sentry Browser SDK Backend. */
 export class BrowserBackend implements Backend {
-  /** Handle to the SDK client for callbacks. */
-  private readonly client: Client<BrowserOptions>;
-
   /** Creates a new browser backend instance. */
-  public constructor(client: Client<BrowserOptions>) {
-    this.client = client;
-  }
+  public constructor(private readonly options: BrowserOptions) {}
```

Comment:
We probably also want the DSN here.

Reply:
I was thinking that we only pass in the options and if we need some fancy behavior our `DSN` class supports, we create a DSN object in the backend. 

See: https://github.com/getsentry/raven-js/pull/1323/files#diff-261a7b228c5d644e74c94c3f99ba8a9eR147

Reply:
WFM. We can assume that the DSN must be valid at that point, so there's also no need to catch exceptions anymore.